targets/TARGET_NUVOTON/TARGET_M451/spi_api.c

```diff
@@ -60,6 +60,34 @@ static struct nu_spi_var spi2_var = {
 #endif
 };
 
+/* Synchronous version of SPI_ENABLE()/SPI_DISABLE() macros
+ *
+ * The SPI peripheral clock is asynchronous with the system clock. In order to make sure the SPI
+ * control logic is enabled/disabled, this bit indicates the real status of SPI controller.
+ *
+ * NOTE: All configurations shall be ready before calling SPI_ENABLE_SYNC().
+ * NOTE: Before changing the configurations of SPIx_CTL, SPIx_CLKDIV, SPIx_SSCTL and SPIx_FIFOCTL registers,
+ *       user shall clear the SPIEN (SPIx_CTL[0]) and confirm the SPIENSTS (SPIx_STATUS[15]) is 0
+ *       (by SPI_DISABLE_SYNC here).
+ */
+__STATIC_INLINE void SPI_ENABLE_SYNC(SPI_T *spi_base)
```

Comment:
static inline should do the job here (all compilers)

Reply:
@0xc0170 Could you give more details?

Reply:
@0xc0170 I don't know your point. Could you explain it?

Reply:
`static inline` should do the job. I believe this macro comes from CMSIS that is also valid so no biggy

Reply:
If you want to update your PR , let us know. We shall trigger CI asap

Reply:
@0xc0170 Yes, `__STATIC_INLINE` comes from CMSIS. I won't make change about it. Just trigger CI.